base/base32768_body.hpp

```diff
@@ -17,22 +19,105 @@ namespace base {
 
 namespace internal_base32768 {
 
-constexpr std::int64_t bits_per_byte = 8;
-constexpr std::int64_t bits_per_code_point = 15;
-constexpr std::int64_t bits_per_final_code_point = 7;
-constexpr std::int64_t bytes_per_code_point =
-    (bits_per_code_point + 2 * bits_per_byte - 2) / bits_per_byte;
-static_assert(bytes_per_code_point == 3,
-              "End of input padding below won't be correct");
+// Ceiling log2 of n.  8 -> 3, 7 -> 2.
```

Comment:
Use `⌈` and `⌉` (what has *A Programming Language* ever done for us?), so `⌈log₂ n⌉`.